baker/SiteBaker.tsx

```diff
@@ -356,6 +380,16 @@ export class SiteBaker {
                 .map((gdoc) => gdoc["featured-image"])
                 .filter((filename): filename is string => !!filename)
 
+            const linkedGrapherCharts = pick(
+                this._prefetchedAttachmentsCache.linkedCharts.graphers,
+                linkedGrapherSlugs
+            )
+            const linkedIndicatorIds = excludeUndefined(
```

Comment:
I think the difference between linked charts and linked indicators might be worth explaining in a comment, maybe in the types file?

If you annotate it with 
```ts
/**
* A comment like this (with two starting asterisks) it will show up when you hover/explain the type
*/
export type LinkedIndicator = {
 // blah
}
```

![image](https://github.com/owid/owid-grapher/assets/11844404/37fd2410-8937-4cb0-9a43-6dca0eac4fb6)